### PR TITLE
[docs] Add WIP pages for the charts

### PR DESCRIPTION
**Pull request by alexfauquette**

### CODE REVIEW

**Review by flaviendelangle — COMMENTED** [review]
https://medium.com/vlearn-together/12-most-common-types-of-charts-in-data-visualisation-36021bd2fef5

I think we can add "Column chart" and "Area chart" (nivo calls it "Stream", chart.js "Area", echarts merges it with "Line").


The other I'm seeing are probably less used.
It can still be useful to add them, just to see if we are underestimating the traction for a set of charts (Funnel, Heatmap, Tree Map, Candle Stick, ...).

**Comment by flaviendelangle** [comment]
> For "Columns" I do not see the distinction with the "Bars". It's the same but rotated by 90°

Most library seem to indeed call both "Bar", let's forget about "Column" then

**Comment by alexfauquette** [comment]
Also added Radar since it seems very common between libraries

File: docs/data/charts/areas/areas.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+# Charts - Areas
+
+<p class="description">Chart areas can express qualities about data, such as highlights, and comparisons.</p>
+
+> ⚠️ This feature isn't implemented yet. It's coming.
+>
+> 👍 Upvote [issue #7928](https://github.com/mui/mui-x/issues/7928) if you want to see it land faster.
+>
+> 💬 To have a solution that solves your needs, leave a comments on the [same issue](https://github.com/mui/mui-x/issues/7928).
```
Comment (flaviendelangle) [inline docs/data/charts/areas/areas.md:14]:
```suggestion
> 💬 To have a solution that meets your needs, leave a comment on the [same issue](https://github.com/mui/mui-x/issues/7928).
```

```suggestion
> 💬 To have a solution that meets your needs, leave comments on the [same issue](https://github.com/mui/mui-x/issues/7928).
```

Or another proposal:

```suggestion
> 👍 Upvote [issue #7928](https://github.com/mui/mui-x/issues/7928) if you want to see it land faster or comment  it make sure the new feature meets your needs.

**Review by flaviendelangle — COMMENTED** [review]
Would it make sense to add a small image of each chart to make sure people understand what it is ?
I did not know the name `Sankey` for example but I recognized the chart after looking for it on Google.

File: docs/data/charts/areas/areas.md (hunk 1 above)
Comment (alexfauquette) [reply docs/data/charts/areas/areas.md]: I like the idea of adding images, but I don't know which one to display. We could use those from material design article but I did not found licence of those images

https://m2.material.io/design/communication/data-visualization.html#types

**Comment by joserodolfofreitas** [comment]
If the goal is to get people to see the issues for upvoting, I believe we should consider having them on a single page.
I don't think it'll be a good experience to navigate page by page to find the issues for upvoting.

**Pull request merged**